_drafts/2021-05-04-cultivating-human-connection.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Cultivating human connection in online learning
+author: Emma Raupp, Abby Henry
+---
+
+As digital classrooms became the “new normal” in lieu of physical classrooms during the pandemic, many social aspects of learning took a hit. Without in-person meetings, many students feel isolated from their peers and instructors. Feeling disconnected from a shared learning environment lowers our motivation to participate and keeping up with coursework quickly becomes a struggle. There are many factors in this widespread sense of burn-out-- a decline in mental health, political stressors, and family crises are just a few that come to mind-- but one factor with a feasible solution is the lack of genuine human connection in digital learning environments. We don’t want students to feel alone if they’re struggling, but the unfortunate reality is that they often do. One solution to this problem is fostering a stronger human connection between instructors and students, in the hope of creating a more supportive digital learning community. 
```

Comment:
I'd replace your " -- " here with an em-dash. On a Mac, you can create an em-dash by pressing **option, shift, dash (-)**.